src/components/LazyLog/index.jsx

```diff
@@ -182,6 +186,19 @@ export default class LazyLog extends Component {
      * Flag to enable/disable case insensitive search
      */
     caseInsensitive: bool,
+    /**
+     * If true, capture system hotkeys for searching the page (Cmd-F, Ctrl-F,
+     * etc.)
+     */
+    captureHotKeys: bool,
+    /**
+     * If true, search like a browser search - enter jumps to the next line
+     * with the searched term, shift + enter goes backwards.
+     * Also adds up and down arrows to search bar to jump
+     * to the next and previous result.
+     * Defaults to false, which causes enter to toggle the filter instead.
+     */
+    searchLikeBrowser: bool,
```

Comment:
I guess my goal has been to keep the base component as close to the original as possible and add extra options in case anyone else were to get use of of this. I can make it default to true for sure, but I guess I don't see the harm of leaving it in?